scripts/darwin.sh

```diff
@@ -61,6 +61,7 @@ for disk in $DISKS; do
     echo "mountpoints:"
     echo "$mountpoints" | while read -r mountpoint ; do
       echo "  - path: \"$mountpoint\""
+      echo "    volumeName: \"$mountpoint\""
```

Comment:
On mac, `path` already has the name of the device `/Volumes/Device_115`.
Not sure what's the best here, probably to parse the string and get `Device_115`.